src/components/organisms/TopNavigation/TopNavigation.jsx

```diff
@@ -7,76 +7,81 @@ import CartMenu from "../../molecules/CartMenu/CartMenu";
 import SubNavigation from "../../molecules/SubNavigation/SubNavigation";
 
 const TopNavigation = (props) => {
-  const { topnav } = props;
-  const { menu } = props;
+  const { TopNav } = props;
```

Comment:
Use comma and put everything in 1 string

Reply:
Done